tests/integration/profiling/rule_based_profilers/test_profiler_user_workflows.py

```diff
@@ -2078,3 +2080,93 @@ def test_quentin_expect_column_unique_value_count_to_be_between_auto_yes_default
 
         max_value_actual: int = result.expectation_config["kwargs"]["max_value"]
         assert max_value_expected - 1 <= max_value_actual <= max_value_expected + 1
+
+
+@pytest.mark.skipif(
+    version.parse(np.version.version) < version.parse("1.21.0"),
+    reason="requires numpy version 1.21.0 or newer",
+)
+@freeze_time("09/26/2019 13:42:41")
+def test_quentin_expect_column_proportion_of_unique_values_to_be_between_auto_yes_default_profiler_config_yes_custom_profiler_config_no(
+    quentin_columnar_table_multi_batch_data_context,
+    set_consistent_seed_within_expectation_default_profiler_config: Callable,
+) -> None:
+    context: DataContext = quentin_columnar_table_multi_batch_data_context
+
+    result: ExpectationValidationResult
+
+    suite: ExpectationSuite
+
+    expectation_suite_name: str = f"tmp.profiler_suite_{str(uuid.uuid4())[:8]}"
+    try:
+        # noinspection PyUnusedLocal
+        suite = context.get_expectation_suite(
```

Comment:
@cdkini Could you please look into the viability of using `get_validator_with_expectation_suite()` (and/or `get_or_create_expectation_suite()` which it utilizes) -- examples are in the same module.  Thanks!